### ISSUE TITLE
The sample  is in the VCF but not assigned in the SETS.txt

### ISSUE THREAD

**Issue by ScooLida** [millanek/Dsuite#82]
Hi!
I can't understand.  I have file like this.
Why Dsuite can't find sample. SAmple is in same directory as SETS.txt file
1k.sort.bam     tanaicus
5k.sort.bam     tanaicus
3k.sort.bam     tanaicus
SRR10012547.sort.bam    europaeus
SRR17908653.sort.bam    europaeus
SRR12518920.sort.bam    europaeus
SRR17908654.sort.bam    europaeus
SRR10011655.sort.bam    europaeus
SRR6485265.sort.bam     americanus
SRR11020211.sort.bam    americanus
..
/mss_users/ltursunova/cow/new2/och.fasta    Outgroup

Error
Did not find the sample: "mss_users/ltursunova/cow/new2/och.fasta"
1k,5k,SRR10012547,SRR11020211,SRR17044867,SRR17908655,SRR5949621,SRR5949630,SRR6485240,SRR6485284,3k,SRR10082089,SRR11020300,SRR17908653,SRR5949623,SRR5949632,SRR10011655,SRR10082097,SRR12518920,SRR17908654,SRR17908659,SRR5949624,SRR5949634,SRR6485281
Did not find any samples in the VCF for "Outgroup"
Dsuite: Dsuite_common.cpp:28: void SetInformation::linkSetsAndVCFpositions(const std::vector<std::__cxx11::basic_string<char> >&): Assertion `!spPos.empty()' failed.
Aborted (core dumped)

**Comment by wuyh0328** [comment]
I have the same problem. Have you solved it?

**Comment by ScooLida** [comment]
> I have the same problem. Have you solved it?

Yes, there is two mistake. Outgroup also must be in .bam file, not .fasta.
And I don't have to write .sort.bam. For list number and group is enough.

1k    tanaicus
5k    tanaicus

**Comment by wuyh0328** [comment]
But I don't have either of those mistakes. It still can't find a sample. Do you know how to solve it?
![1703664569973](https://github.com/millanek/Dsuite/assets/150041260/f5a3e702-85ba-4bc0-94c7-02ad88951e59)

**Comment by ScooLida** [comment]
> But I don't have either of those mistakes. It still can't find a sample. Do you know how to solve it? ![1703664569973](https://private-user-images.githubusercontent.com/150041260/292988891-f5a3e702-85ba-4bc0-94c7-02ad88951e59.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM2NjQ5NjUsIm5iZiI6MTcwMzY2NDY2NSwicGF0aCI6Ii8xNTAwNDEyNjAvMjkyOTg4ODkxLWY1YTNlNzAyLTg1YmEtNGJjMC05NGM3LTAyYWQ4ODk1MWU1OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyN1QwODExMDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNTA4YjM0NTUwODEwMjYwMDUyYzBiZTBlNDhjOTM5MzQxOWE4ZTJjZWExYmIzZGU0OTZjYTUxMjMwZWZmYzgzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.0zNaJ2wnI_jnNBnbAy5OfbPuBIIWUb1-pP8QBM0ZebQ)

Do you have this sample at all(MDe2A)? Cheсk your data

**Comment by wuyh0328** [comment]
Thank you! YES,You can see it in the sample list below。(I use the module Dquartets).

**Comment by ScooLida** [comment]
> Thank you! YES,You can see it in the sample list below。(I use the module Dquartets).

May be MDe2A appears twice in sets. There's nothing more I can help

**Comment by wuyh0328** [comment]
Ok, thank you very much. I will check again.

**Comment by wuyh0328** [comment]
> > Thank you! YES,You can see it in the sample list below。(I use the module Dquartets).
> 
> May be MDe2A appears twice in sets. There's nothing more I can help

I seem to have found the problem. In my SETS.txt file, there are Spaces after the sample ID.
